README.md

```diff
@@ -9,14 +9,14 @@ A GitLab API client enabling Go programs to interact with GitLab in a simple and
 
 ## NOTE
 
-Release v0.6.0 (released on 25-08-2017) no longer supports the older V3 Gitlab API. If
+Release v0.6.0 (released on 25-08-2017) no longer supports the older V3 GitLab API. If
 you need V3 support, please use the `f-api-v3` branch. This release contains some backwards
-incompatible changes that were needed to fully support the V4 Gitlab API.
+incompatible changes that were needed to fully support the V4 GitLab API.
 
 ## Coverage
 
-This API client package covers most of the existing Gitlab API calls and is updated regularly
-to add new and/or missing endpoints. Currently the following services are supported:
+This API client package covers most of the existing GitLab API calls and is updated regularly
+to add new and/or missing endpoints. Currently, the following services are supported:
```

Comment:
I'm fine with updating the name GitLab, but are you 100% sure this comma _needs_ to be here? Or is it optional depending on the rest of the sentense?


As to me it reads a bit strange this way, yet I'm a non-native English speaker so I'm not positive.

Reply:
When "currently" is the introductory word at the beginning of a sentence, you need a comma after it. [See more](https://linguaholic.com/linguablog/comma-after-currently/)

Reply:
OK, I'll take it... Thanks.